.changes/next-release/bugfix-EndpointProvider-38157.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "EndpointProvider",
+  "description": "Addresses issue `#2938 <https://github.com/boto/botocore/issues/2938>`__"
```

Lets provide a clear message on the issue and then we can link.

```suggestion
  "description": "Fixed bug in virtual addressing for S3 Buckets `#2938 <https://github.com/boto/botocore/issues/2938>`__"
```